### DISCUSSION TITLE
Exception: Module _errorhandling could not be loaded.

### DISCUSSION THREAD

**Discussion by 0mniSci** [Monika-After-Story/MonikaModDev#9714] [QA - tech support + general]
So I had an issue w/ the updater. After following the directions on the FAQ, I got the updater to work, but now it's telling me that Module_errorhandling could not be loaded. Here is my traceback

[traceback.txt](https://github.com/Monika-After-Story/MonikaModDev/files/9905667/traceback.txt)

**Comment by Booplicate** [comment]
You have incomplete renpy binaries, make a fresh install of DDLC and MAS
